index.html

```diff
@@ -0,0 +1,151 @@
+          Leda Sobierajski, together the dynamic designer couple run a design
+          studio.
+        </p>
+      </div>
+    </section>
+    <div>
+      <section class="talks" id="talks">
+        <h2>Talks</h2>
+        <h3>Strategy: Designing a Small Business</h3>
+        <h4>Presented by Leta Sobierajski & Wade Jeffree</h4>
+        <p>
+          Description: Marketing plans, financial reports and contracts are not
+          what creative dreams are made of. But they are an integral part of any
+          small business. How do you achieve success establishing and
+          maintaining a small design studio? How do you define success over the
+          short and long-term of a studio’s existence? Join three sets of owners
+          of acclaimed creative boutique as they describe their journeys and
+          offer insights from what they’ve learned along the way and then answer
+          audience questions.
+        </p>
+        <h3>Content Design: Why There's Another Title in Our Industry</h3>
+        <h4>Presented by Sarah Richardson</h4>
+        <p>
+          Copywriter, UX writer, technical author, web editor... it's slightly
+          ironic an industry of wordsmiths have so many titles for themselves.
+          Then, in 2011 Sarah Richards created the term 'Content Design' for the
+          Government Digital service and added another term to the industry
+          lexicon. It caused a lot of consternation in government at the time.
+          Which was the point. The reason was quite clear: government had to
+          change the way it considered the skill of communicating. GOV.UK was
+          not mucking around. Sarah will explain why changing the name was
+          important at the time, the process of content design, the impact it
+          has and how it has changed content production around the world. Sarah
+          will share examples of how a single word can stop a multi-million
+          pound service dead in its tracks, why you should have a content person
+          at discovery and how content can make or break service design.
+        </p>
+        <h3>Design for the Modern Ages</h3>
+        <h4>Presented by Professor Elizabeth Guffrey</h4>
+        <p>
+          Professor Guffey will be delivering a presentation on design as it has
+          evolved throughout not only her career but also the farther past.
+          Ending the presentation with a topic of how designers can change today
+          to adapt to modern design practices, etc. How can we adapt design to
+          be more inclusive and considerate of separate groups? How can design
+          be used to help improve our society, and more.
+        </p>
+      </section>
+    </div>
+    <footer class="call"> Remember! we desire nothing but to inspire designers everywhere to create a better world, go forth and design! contact us at: Designmodernconference@gmail.com </footer>
```

Comment:
You could add an `a` element here to make the email address clickable/functional:

```suggestion
    <footer class="call"> Remember! we desire nothing but to inspire designers everywhere to create a better world, go forth and design! contact us at: <a href="mailto:designmodernconference@gmail.com">Designmodernconference@gmail.com</a></footer>
```